jetty-server/src/main/java/org/eclipse/jetty/server/HttpOutput.java

```diff
@@ -1044,7 +1044,7 @@ public void run()
                             if (LOG.isDebugEnabled())
                                 LOG.debug("onError", th);
                             _writeListener.onError(th);
-                            close();
+                            abort(th);
```

Comment:
oof.  Big change in HttpOutput.

@gregw this change needs your review.

Reply:
@joakime indeed :)   Simone and I hung out with Lachlan on this one... it was my proposal to which Simone said "oof big change!".    We think it is good, but want to see a full build pass and to ponder a bit more.

But basically, if a pending write times out, you are dead.  Nothing you can do to get the write back or cancel it or write a sane response.  Abort is there to signal to the client as best as possible that the response is cactus!

Reply:
only seems to have broken `testAsyncWriteThrowsException` and `testAsyncWriteThrowsError` in `org.eclipse.jetty.http.client.AsyncIOServletTest`